### PR TITLE
feat: otomi branding for Keycloak (PoC)

### PR DESCRIPTION
**Pull request by j-zimnowoda**
It used to be an open PR and for some reason we lost it, so I am opening it again.

### CODE REVIEW

File: values/keycloak/keycloak.gotmpl [hunk 1]
```diff
@@ -31,6 +31,39 @@ keycloak:
       memory: 1Gi
     {{- end }}
   replicas: 1
+  startupScripts:
+    install-theme.sh: |
+      #!/bin/sh
+      cp -Rv /theme/otomi /opt/jboss/keycloak/themes/
+  extraInitContainers: |
+    - name: keycloak-otomi-branding
+      image: "cdrocker/keycloak-otomi-branding:0.1.1"
```
Comment (j-zimnowoda) [inline values/keycloak/keycloak.gotmpl:40]: need to be pulled from our repo

**Comment by Morriz** [comment]
It's not much. Just 2 lines loading somebody else's theme

File: values/keycloak/keycloak.gotmpl (hunk 1 above)
Comment (Morriz) [inline values/keycloak/keycloak.gotmpl:40]: cdrocker?

I thought we would put it in the `otomi/console` image

File: values/keycloak/keycloak.gotmpl [hunk 2]
```diff
@@ -31,6 +31,39 @@ keycloak:
...
+          cpu: 50m
+          memory: 16Mi
+        limits:
+          cpu: 500m
+          memory: 16Mi
+      volumeMounts:
+        - name: theme
+          mountPath: /theme
+
+  extraVolumes: |
```
Comment (Morriz) [inline values/keycloak/keycloak.gotmpl:60]: why are these pipes at the end?

File: values/keycloak/keycloak.gotmpl [hunk 3]
```diff
@@ -31,6 +31,39 @@ keycloak:
...
+    - name: theme
+      emptyDir: {}
+
+  extraVolumeMounts: |
```
Comment (Morriz) [inline values/keycloak/keycloak.gotmpl:64]: again, what's with that pipe?

**Review by Morriz — CHANGES_REQUESTED** [review]

**Comment by Morriz** [comment]
@j-zimnowoda can I close this? I see another PR open for this

**Comment by j-zimnowoda** [comment]
> @j-zimnowoda can I close this? I see another PR open for this

@Morriz I am working on this PR

**Comment by Morriz** [comment]
of course ;p

**Review by Morriz — CHANGES_REQUESTED** [review]
please remove those redundant pipes

**Review by Morriz — APPROVED** [review]

**Comment by j-zimnowoda** [comment]
> of course ;p
> 
> still interested to know why those pipes were put there ;)

@Morriz I also did not like those pipes. Bleeee
But after removing them helm could template it anymore.
It is because they expected extraInitcontainer to be raw string which they can template

**Pull request merged**